code/switch.js

```diff
@@ -0,0 +1,117 @@
+// Start here
+
+// Step 1 - Welcome and introduction
+alert(
+    `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+);
+
+//Ask user for their name using prompt method (when using conts the variable name doesn't appear in the alert. it says that it's deprecated)
+const userName = prompt(`What's your name? - ex. Maria`);
+
+//Display a greeting to confirm users name
+alert(`Hi ${userName}! Welcome to Javascript Pizzeria.`);
+
+
+// Step 2 - Food choice
+//In this .js file version I would like to explore how to use objects/arrays to be used combined with a switch condition
+//I think that using objects (probably an array would be just fine here as well) is a logical way of organizing values
+
+//Define dish objects (type and subtype)
+const dishes = {
```

Comment:
Great that you try object for the purpose of learning! To simplify the codes, I would suggest trying an array inside each of the category instead as you have the same key ("name") for every value. E.g. pizza: ["Margherita", "Pepperoni", "Gorgonzola"]